src/main/java/cz/cvut/kbss/analysis/service/UserRepositoryService.java

```diff
@@ -63,12 +63,12 @@ public void updateCurrent(UserUpdateDTO userUpdate) {
         User currentUser = getCurrentUser();
         if (!currentUser.getUri().equals(userUpdate.getUri())) {
             log.warn("< updateCurrent - URIs do not match! {} != {}", currentUser.getUri(), userUpdate.getUri());
-            throw new LogicViolationException("User update uri does not match current user!");
+            throw new LogicViolationException("errors.user.update.uriMismatch","User update uri does not match current user!");
```

Comment:
```suggestion
            throw new LogicViolationException("error.user.update.uriMismatch","User update uri does not match current user!");
```

Reply:
Everywhere else you have `error` but here `errors`

Reply:
done!